ntc_templates/templates/ubiquiti_edgerouter_show_interfaces.textfsm

```diff
@@ -0,0 +1,25 @@
+Value Filldown INTERFACE (\w+)
+Value DESCRIPTION ([\w\d\-]+)
+Value IPV4_ADDRESS ([\d\.]+/\d+)
+Value IPV6_ADDRESS ([\d\w\:]+/\d+)
+Value Filldown STATE (\w)
+Value Filldown LINK_STATUS (\w)
+
+Start
+  ^${INTERFACE}\s+${IPV4_ADDRESS}\s+(${STATE}/${LINK_STATUS})\s+${DESCRIPTION} -> Continue
+  ^${INTERFACE}\s+${IPV6_ADDRESS}\s+(${STATE}/${LINK_STATUS})\s+${DESCRIPTION} -> Record
+  ^\s+${IPV4_ADDRESS} -> Continue
+  ^\s+${IPV6_ADDRESS} -> Record
+  ^${INTERFACE}\s+-\s+(${STATE}/${LINK_STATUS})\s+${DESCRIPTION} -> Record
+  ^\s+${IPV4_ADDRESS} -> Continue
+  ^\s+${IPV6_ADDRESS} -> Record
+  ^${INTERFACE}\s+${IPV4_ADDRESS}\s+(${STATE}/${LINK_STATUS}) -> Continue
+  ^${INTERFACE}\s+${IPV6_ADDRESS}\s+(${STATE}/${LINK_STATUS}) -> Record
+  ^\s+${IPV4_ADDRESS} -> Continue
+  ^\s+${IPV6_ADDRESS} -> Record
+  ^${INTERFACE}\s+${IPV4_ADDRESS}\s+(${STATE}/${LINK_STATUS}) -> Continue
+  ^${INTERFACE}\s+${IPV6_ADDRESS}\s+(${STATE}/${LINK_STATUS}) -> Record
+  ^\s+${IPV4_ADDRESS} -> Continue
+  ^\s+${IPV6_ADDRESS} -> Record
```

Comment:
There are way too much possibilities.
You should be able to do something much cleaner with "Continute.record" like in huawei_vrp_display_acl_all
You can use also "Required" to indicate the value need to be collected before recording.

Those 15 lines can be reduced to 3 or 5

You can also check huawei_vrp_display_interface_description for that thing of Starting the recording only after a certain line. You would ensure you would start recording right after
"Interface    IP Address                        S/L  Description "
and you would be able to include the first lines